case-studies/bayesline.md

```diff
@@ -90,17 +96,19 @@ When embarking on building Bayesline, Sebastian and his co-founder wanted an ope
 So we basically need the tool that gets us to the finish line fastest without having to learn (a new framework) and without a super steep learning curve.
 ```
 
-Sebastian and his team built an initial prototype of Dash, a low-code framework for rapidly building data apps in Python.
+Sebastian and his team built an initial prototype with Dash, a low-code framework for rapidly building data apps in Python.
 As their app grew, it eventually became slow and difficult to maintain.
 
 ```md quote Sebastian
-The app was just getting painfully slow. Dash loads the entire application, the entire dom of every single page...by definition, almost as the application gets bigger, the performance will just grind, will go down.
+The app was just getting painfully slow. Dash loads the entire application, the entire dom of every single page... as the application gets bigger, the performance will just go down.
 ```
 
+## How Bayesline built a production-grade web app with Reflex
```

Comment:
How Reflex allowed Bayesline to build faster during Y Combinator

Reply:
feel it's important to keep the production grade word in here, let me know what you think of the change